SCons/Taskmaster.py

```diff
@@ -230,8 +232,8 @@ def execute(self):
                 for t in cached_targets:
                     try:
                         t.fs.unlink(t.get_internal_path())
-                    except (IOError, OSError):
-                        pass
+                    except (IOError, OSError) as e:
+                        display("scons: failed to delete partial cache file %s: %s" % (t.get_internal_path(), e))
```

Comment:
Technically you're just outputting a message and neither issuing a SCons warning or error. 
Is the intent that if this should happen that SCons should stop executing at that point? or just issue a warning and continue?

Reply:
IMO, this is pretty bad because it means some file may exists even when the build expects it not to. I generally prefer hard fails, to prevent cascading and overlooked issues.

but I didn't want a PR that changed functionality, so I just went with a log message. I can switch to a warning if that is preferred.

Reply:
we haven't implemented anything in our vendored scons around this yet

Reply:
switched to warning in 9dc6cbe

Reply:
I'd probably vote for a warning, but this falls into one of those "we don't know what to do here, so just ignore" cases that there are a few of, so anything is an improvement.